src/renderer/react/my-crypto/containers/crypto-price-table.js

```diff
@@ -9,7 +9,8 @@ const mapStateToProps = state => {
 	return {
 		...getLocale(state),
 		...getFiatCurrency(state),
-		tokens: getTokens(state)
+		tokens: getTokens(state),
+		alwaysVisible: ['', '0x4cc19356f2d37338b9802aa8e8fc58b0373296e7']
```

Comment:
@sk91 what about Ethereum? Will it use the same address?

Reply:
@rodrigopavezi Ethereum address is '' -> no address